slink/docker-compose.yml

```
@@ -0,0 +1,25 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: slink_web_1
+      APP_PORT: 3000
+
+  web:
+    image: anirdev/slink:v1.7.3@sha256:4dc9e1849a0a814d51c031f1e3870b9c58767192b3fff4f46a89005879ca0042
+    restart: on-failure
+    environment:
+      TZ: Europe/Berlin
```

`UTC` as the default timezone instead of `Europe/Berlin` it keeps things simpler and avoids daylight saving time issues.
```suggestion
      TZ: UTC
```